ch04/ex-paragraphs.md

```diff
@@ -54,3 +54,29 @@ Any tags that are open when encountering the second paragraph should be closed
          <p>
            <b>
              'world'
+
+
+Test when `<p>` tags have different attributes from each other
+
+    >>> test_parse("<p>This is a paragraph with <p>another nested paragraph inside.</p></p>")
```

Comment:
Don't put the closing `</p></p>` tags

Reply:
Also these `<p>` tags do not have different attributes from one another. They don't have any attributes at all!